runtime/cmd/decode-state-values/main.go

```diff
@@ -136,6 +141,11 @@ type slabStorage struct{}
 
 var _ atree.SlabStorage = &slabStorage{}
 
+func (s *slabStorage) RetrieveIfLoaded(atree.SlabID) atree.Slab {
+	// RetrieveIfLoaded() is used for loaded resource tracking.  So it isn't needed here.
+	panic(runtimeErr.NewUnreachableError())
```

Comment:
Is this a fatal error that we have to stop the execution node, or it might mean that user tx has to fail and move on?

Reply:
`UnreachableError` is an implementation error. Still, we recover from implementation errors gracefully and abort transactions ("treat them like user errors"), so this won't bring down nodes